docker-compose.yml

```diff
@@ -0,0 +1,43 @@
+version: '2'
+services:
+  NRESCeleryRabbitmq:
+    image: rabbitmq:3-management
+    ports:
+    - 15673:15672/tcp
+    labels:
+      io.rancher.container.pull_image: always
+  NRESPipelineTestFitsExchange:
+    image: rabbitmq:3-management
+    ports:
+    - 15674:15672/tcp
+    labels:
+      io.rancher.container.pull_image: always
+  NRESPipelineE2EData:
+    image: docker.lco.global/nres-pipeline-e2e-data:1.0.0
+    entrypoint:
+    - /bin/true
+    volumes:
+    - nrese2e:/archive
+    labels:
+      io.rancher.container.pull_image: always
+      io.rancher.container.start_once: 'true'
+  NRESPipelineTest:
+    image: ${DOCKER_IMG}
+    environment:
+      BOSUN_HOSTNAME: alertsdev.lco.gtn
+      BROKER_PASSWORD: guest
```

Comment:
Credentials should be in the Jenkins credentials store.

Reply:
Done.